src/main/java/seedu/momentum/logic/Logic.java

```diff
@@ -37,6 +39,11 @@
      */
     ObservableList<Project> getFilteredProjectList();
 
+    /**
+     * Returns the collection of all tags that the user has entered for the projects.
+     */
+    Set<Tag> getProjectTags();
+
```

Comment:
Should this method belong to ReadOnlyProjectBook instead? Logic should not have to know about the internal components of Model (i.e. tags). Can probably get the projectbook from logic, then get the tags from the projectbook

Reply:
Ok yup I agree with this. Will make the changes.